docs/howto/how-to-make-tensorflow-plugin-with-ui/tensorflow-plugin-with-ui.md

```diff
@@ -0,0 +1,1488 @@
+# 1. Create a Basic Text Classifier Neural Network with TensorFlow
```

Comment:
TW review: I recommend removing the numbering in the headings. Instead of 1. Create a Basic Text Classifier Neural Network with TensorFlow, update to:

Create a Basic Text Classifier Neural Network with TensorFlow

And the same for all headings that follow.